components/tooltip/README.md

```diff
@@ -163,15 +193,24 @@ The `d2l-tooltip-help` component is used to display additional information when
 <!-- docs: end donts -->
 <!-- docs: end best practices -->
 
-<!-- docs: demo live name:d2l-tooltip-help autoSize:false size:small -->
+<!-- docs: demo live name:d2l-tooltip-help autoSize:false size:medium -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/tooltip/tooltip-help.js';
 </script>
-
-<p class="d2l-body-standard">
+<!-- docs: start hidden content --> 
+<script>
+  window.addEventListener('load', function () {
+    setTimeout(function() {
+      const tooltip = document.querySelector('d2l-tooltip-help[text="Helpful label"]:not(#help-tooltip)');
```

Comment:
So I realized that we don't actually need to stress with selectors here, and can literally just have 
`const tooltip = document.querySelector('d2l-tooltip-help');` or 
`const tooltip = document.querySelector('d2l-tooltip');` for all the demos and still have this code work (the tooltips auto-show on page load). 

Do we want to do that? Or would it be potentially confusing for someone reading the raw README?

Reply:
That's because they're all iframed ya, so not colliding with each other?

Reply:
Yup [looks like it](https://github.com/BrightspaceUI/documentation/blob/main/pages/_components/demo-snippet.js#L67-L68). 

In terms of readability (someone reading the raw README), are there any concerns? 
Or should I just make the changes to simplify the selector queries?

Reply:
I'd go for simplicity of the selectors and wouldn't worry about raw README readers haha.